src/test/java/api/requests/RequestsAPIRetrievalTests.java

```diff
@@ -373,6 +374,17 @@ public void canGetMultipleRequests()
     requestList.forEach(this::requestHasServicePointProperties);
     requestList.forEach(this::requestHasPatronGroupProperties);
     requestList.forEach(this::requestHasTags);
+
+    requestHasCallNumberStringProperties(requestList.stream()
+        .filter(req -> nod.getId().toString().equals(req.getString("itemId")))
+        .findFirst()
+        .orElseThrow(() -> new AssertionError("Expected Nod in response")),
+      "nod");
```

Comment:
Could a property be extracted to express the intent of this usage of streams, e.g. `findRequestByItemId`?